text/0000-crate-name-transfer.md

```diff
@@ -0,0 +1,447 @@
+member should be available to attend crates.io team meetings to discuss the
+subteam's activity. Not necessarily the same member every time, and not
+necessarily every week; we can pin this down once things are up and running and
+we get a sense of the scale of incoming requests. Around the 6 month mark if the
+experiment has made it that far, the crates.io team should meet with the full
+Responsible Party to check in and dig into how our processes can be improved.
+
+To reiterate: crate transfers are to happen purely by group decision of the
+Responsible Party. They are not bound by any checklist of criteria specified by
+this eRFC. The Responsible Party are entrusted with upholding the intent of the
+eRFC in performing transfers in only the most clear-cut cases according to their
+judgement.
+
+When a user requests ownership of a crate, the two common responses by the
+Responsible Party will be:
+
+1. All right, we have yanked the existing releases and set you as the owner!
+   Please ensure that the next version you publish is semver-incompatible with
+   any of the old yanked releases.
+
+2. Hi! Thanks for the request. We believe this case is less clear-cut than the
+   ones we are currently willing to grant. We would recommend picking a
+   different crate name for now, but we can leave this request open if you want
+   and revisit the decision in the future.
+
+The Responsible Party should feel comfortable reaching these decisions
+privately, but are to maintain some sort of public log of how many requests were
+made, how many were rejected, and some brief discussion of the rationale of the
+accepted requests (and they must be brief, because it's supposed to be clear
+cut). The general principle is to strive to be as transparent as possible
+without interfering with the team's ability to get work done.
+
+When accepting a request, the Responsible Party will reach out to an operations
+person on crates.io who will perform the transfer. There is a possibility that
+this could be automated later but this is not required for the eRFC to go
+forward; some design and implementation and policy work would be involved.
+
+# Guide-level explanation
+
+The details of requesting ownership of a crate name are to be figured out over
+time, but a viable way to begin would be as follows.
+
+- Set up a repository under https://github.com/rust-lang called
+  "crate-transfers".
+
+- Write a readme outlining the process. Ensure that the readme conveys that only
+  the most clear-cut requests will be granted, so not to expect too much.
+
+- Requests are made by filing a GitHub issue.
+
+- Include a list of "up for grabs" crate names in the readme. Authors that no
+  longer wish to use a crate name that they own can add it to the list by filing
+  an issue or sending a pull request. A crate that is "up for grabs" trivially
+  meets the "clear-cut situation" requirement if ownership of that name is ever
+  requested.
```

Comment:
It might be better to manage this with issues as well, maybe having labels `up-for-grabs` and `transfer-request`. I'm worried that updating the readme has more friction, and it could get out of sync.